packages/widgets/src/components/Carousel.tsx

```diff
@@ -29,26 +29,24 @@ interface CarouselProps {
 
 // TODO: change to slick grid
 export class CarouselComponent extends React.Component<CarouselProps> {
-  divRef: HTMLElement | null = null;
-  sliderRef: Slider | null = null;
+  divRef = React.createRef<HTMLDivElement>();
```

Comment:
Is this part of the fix or just some refactoring?

Reply:
Just refactoring